Mathlib/Algebra/Order/Interval/Basic.lean

```diff
@@ -657,16 +667,19 @@ open Lean Meta Qq
 /-- Extension for the `positivity` tactic: The length of an interval is always nonnegative. -/
 @[positivity NonemptyInterval.length _]
 def evalNonemptyIntervalLength : PositivityExt where
-  eval {u _α} _ _ e := do
-    let ~q(@NonemptyInterval.length _ $inst $a) := e | throwError "not NonemptyInterval.length"
+  eval {u α} _ _ e := do
+    let ~q(@NonemptyInterval.length _ $ig $ipo $a) := e |
+      throwError "not NonemptyInterval.length"
+    let _i ← synthInstanceQ q(IsOrderedAddMonoid $α)
```

Comment:
Again I don't want to sign off on this part of the PR because I don't understand it.